adlfs/spec.py

```diff
@@ -737,26 +737,31 @@ async def _ls(
         return_glob: bool
 
         """
-        logger.debug(f"abfs.ls() is searching for {path}")
+        logger.debug("abfs.ls() is searching for %s", path)
```

Comment:
FYI, it's depending on how expensive string formatting is, it's better to use `"%"`-style format strings. That avoids calling `str(path)` if the level is higher than debug.